src/olm.cpp

```diff
@@ -1005,4 +1009,36 @@ size_t olm_ed25519_verify(
     );
 }
 
+extern "C" {
+    struct s_mallinfo {
+        int arena;    /* non-mmapped space allocated from system */
+        int ordblks;  /* number of free chunks */
+        int smblks;   /* always 0 */
+        int hblks;    /* always 0 */
+        int hblkhd;   /* space in mmapped regions */
+        int usmblks;  /* maximum total allocated space */
+        int fsmblks;  /* always 0 */
+        int uordblks; /* total allocated space */
+        int fordblks; /* total free space */
+        int keepcost; /* releasable (via malloc_trim) space */
+    };
+
+    extern s_mallinfo mallinfo();
+    extern void* sbrk(intptr_t increment);
+}
+
+unsigned int olm_get_total_memory() {
+    unsigned int total_memory = EM_ASM_INT(return HEAP8.length);
+    std::cout << "Total memory: " << total_memory << " bytes" << std::endl;
```

Comment:
Are we really sure we want to print these directly here? I imagine this might be resistant to our `transform-remove-console` transform on web prod, which would mean users might be seeing these logs if we call these functions for them

Reply:
The purpose of these functions is to gain some metrics, ideally in production too. So without logging added, functions are basically useless. 

I added this here to intentionally bypass `transform-remove-console` to save some work configuring logging on prod keyserver and prod shared worker (according to [ENG-10405](https://linear.app/comm/issue/ENG-10405/macos-app-experiencing-wasm-memory-issues-again) we're facing the same issues on web/desktop app) - but if you think it better to avoid logging here and handle it on Comm codebase side I am okay with this too. Let me know what you prefer

Reply:
Hmm... I guess this is okay, I won't slow you down